src/containers/Chat/index.tsx

```diff
@@ -146,11 +146,34 @@ export class Component extends React.Component<ComponentProps, {}> {
     }
 
     public async navigationButtonPressed({ buttonId }: any) {
-        if (buttonId === 'delete') {
+        const { conversation, session } = this.props;
+
+        const conversationalists = conversation.participants.filter(
```

Comment:
You already have `session.username`. Why do you find a user with userId includes a username and make the username again from a userId?

You are doing a lot of unneeded operations. Try to fix your code.